packaging/os/apt.py

```diff
@@ -73,6 +73,12 @@
     required: false
     default: "no"
     choices: [ "yes", "no" ]
+  allow_unauthenticated:
+    description:
+      - Ignore if packages can't be authenticated. This is useful for packages that bootstrap their own apt-key setup.
```

Comment:
IMHO the description is not quite right. If the package is signed, and the key is registered using apt-key, `--allow-unauthenticated` is not needed.

Would you mind change the description to just:
> Ignore if packages can't be authenticated.